docs/_static/js/auto_module_index.js

```diff
@@ -10,15 +10,17 @@ function auto_index(module) {
     var html = "<ul>";
 
     for (var i = 0; i < targets.length; ++i) {
-      var id = $(targets[i]).attr('id');
-      // remove 'mxnet.' prefix to make menus shorter
-      var id_simple = id.replace(/^mxnet\./, '');
-      html += "<li><a class='reference internal' href='#";
-      html += id;
-      html += "'>" + id_simple + "</a></li>";
+	var id = $(targets[i]).attr('id');
+	if ( id ) {
```

Comment:
[minor] remove spaces around `( )`

Reply:
This seems to be a critical PR. Making this change will trigger another 2-3 hour wait before this can be merged. I will hold off this chagne for another PR.